packages/fx-core/src/component/driver/aad/interface/updateAadAppArgs.ts

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+export interface UpdateAadAppArgs {
+  manifestPath: string; // Relative path to project root
+  manifestOutputPath: string; // Relative path to project root
```

Comment:
if the manifestOutputPath is required?

Reply:
I think we can make it optional. If user provides output path (e.g. build/aad.manifest.dev.json), we will write the actual manifest to that path. If not, we do nothing.
@SLdragon @KennethBWSong @jidddddd @adashen how do you think about above proposal? The current behavior in V2 is we always write actual manifest to `build/aad.manifest.{env}.json`.

Reply:
also cc @MuyangAmigo for your thoughts of above proposal

Reply:
I would like to have the output file which is consistent with common build behaviour.  And since the path is relative, where did we get the project root? Do we have a common utility in fxcore for project root?

Reply:
Besides, when user select a custom manifest from external folder. Is it still good to use relative path here?

Reply:
Will create additional PR to compute the actual absolute path because current context does not have project root path yet. Don't want to make this PR pending too much time.